services/forgotPassword.js

```diff
@@ -0,0 +1,35 @@
+const jwt = require("jsonwebtoken");
+const User = require("../db/models/User");
+const sendMailService = require("./sendemail");
+const userRepository = require("../repositories/user");
+const env = require("../configuration/env")
+
+async function setResetPasswordToken(user) {
```

Comment:
same block of code we have in login service. Please create some helper function that will do this part(create token).